core/templates/App.ts

```diff
@@ -151,18 +153,20 @@ angular.module('oppia').config([
                 warningMessage = rejection.data.error;
               }
               AlertsService.addWarning(warningMessage);
-              // rejection.config is an optional parameter.
-              // see https://docs.angularjs.org/api/ng/service/$http
-              var rejectionUrl = typeof rejection.config !== 'undefined' ? (
-                rejection.config.url) : '';
-              var additionalLoggingInfo = warningMessage +
-                '\n URL: ' + rejectionUrl +
-                '\n data: ' + JSON.stringify(rejection.data);
-              // $exceptionHandler is called directly instead of
-              // throwing an error to invoke it because the return
-              // statement below must execute. There are tests
-              // that rely on this.
-              $exceptionHandler(new Error(additionalLoggingInfo));
+              if (!DEV_MODE) {
```

Comment:
Do you actually need this, given that you already have a check in $exceptionHandler? It may be simpler logically to just have a check over there and remove this one.

Reply:
Removed the check here. Done,